images/win/scripts/Tests/Svn.Tests.ps1

```diff
@@ -0,0 +1,6 @@
+
+Describe "Svn" {
```

Comment:
Why we create the separate file for this tool? We group such tests by software report category and put to common file: https://github.com/actions/virtual-environments/blob/master/images/win/Windows2019-Readme.md#tools

Reply:
The same for other tools